README.md

```
@@ -0,0 +1,227 @@
+        TanStack[TanStack Query v5.90.20]
+        Zustand[Zustand v5.0.10]
+        RHF[React Hook Form v7.71.1]
+        ZodFE[Zod v4.3.6]
+        Framer[Framer Motion v12.27.0]
+    end
+
+    subgraph "Backend技術"
+        Hono[Hono v4.11.7]
+        Workers[Cloudflare Workers]
+        Zod[Zod v4.3.6]
+        Bcrypt[bcryptjs]
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)